### DISCUSSION TITLE
Alerts for negative pricing

### DISCUSSION THREAD

**Discussion by lindsay245jpe** [BottlecapDave/HomeAssistant-OctopusEnergy#903] [Ideas]
Hi,
How can we set alerts for negative pricing. I'm gutted, should of been offering to take in the neighbours washing, at times, over the past two days.
There must be a way to set up alerts when negative pricing is announced on the agile tarrif. Does anyone know how to do this as i'm only a cut and paste developer?

thanks

Lindsay

**Comment by BottlecapDave** [comment]
Hello. You could setup an automation which looks at the [current](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/#current-rate) or [next](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/#next-rate) rate sensors (depending on how much notice you want) and send you a notification through your medium of choice. For how to do this, it might be best to ask in the wider HA forums as the true answer depends on how far you want to go.

If this is not enough notice, then you can use a [target rate sensor](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/target_rate/) which a maximum rate of 0, which will turn on when you're in a negative pricing period and provides a schedule up front.
